jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/DefaultProjectAccountSettingsManager.kt

```diff
@@ -49,7 +53,7 @@ class DefaultProjectAccountSettingsManager(private val project: Project) : Proje
                 CredentialManager.getInstance().getCredentialIdentifierById(credentialId)
             }
 
-            val regionId = state.activeRegion ?: AwsRegionProvider.getInstance().defaultRegion().id
+            val regionId = state.activeRegion ?: credentials?.defaultRegion ?: AwsRegionProvider.getInstance().defaultRegion().id
```

Comment:
We somtimes dont have a default region on start even though we have `AwsRegionProvider.getInstance().defaultRegion().id` before

This PR is not aiming to fix that bug is it?

Reply:
I'm not sure I understand your comment... with this you should always get a region - as long as we have a `defaultRegion` specified as part of the `AwsRegionProvider` right?